workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt

```diff
@@ -322,6 +325,18 @@ internal class WorkspaceServiceImpl(
         .map { WorkspaceFile(fileName = it) }
   }
 
+  override fun createSecret(
+      organizationId: String,
+      workspaceId: String,
+      workspaceSecret: WorkspaceSecret
+  ) {
+    val workspaceKey = findWorkspaceById(organizationId, workspaceId).key
+    secretManager.createSecret(
```

Comment:
Could be interesting to add a Map<String,String> in workspaceSecret to allow passing "custom" secret config. Right now, the createSecret endpoint only create `WORKSPACE_EVENTHUB_ACCESSKEY_SECRET`